specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/managedClusters.json

```diff
@@ -3556,9 +3556,90 @@
         "clientId"
       ]
     },
+    "VirtualMachinesProfile": {
+      "type": "object",
+      "properties": {
+        "scale": {
+          "$ref": "#/definitions/ScaleProfile",
+          "description": "Specifications on how to scale a VirtualMachines agent pool."
+        }
+      },
+      "description": "Specifications on VirtualMachines agent pool."
+    },
+    "ScaleProfile": {
+      "type": "object",
+      "properties": {
+        "manual": {
+          "$ref": "#/definitions/ManualScaleProfile",
+          "description": "Specifications on how to scale the VirtualMachines agent pool to a fixed size."
+        }
+      },
+      "description": "Specifications on how to scale a VirtualMachines agent pool."
+    },
+    "ManualScaleProfile": {
+      "type": "object",
+      "properties": {
+        "machines": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManualScaleMachines"
+          },
+          "description": "Specifications on how to scale a VirtualMachines agent pool. Currently only 1 list of machine sizes is allowed."
+        }
+      }
+    },
+    "ManualScaleMachines": {
+      "type": "object",
+      "properties": {
+        "sizes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of allowed vm sizes. AKS will use the first available one when scaling. If a VM size is unavailable (for example due to quota, regional capacity, etc) the next size will be used."
+        },
+        "count": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 1000,
+          "description": "Number of nodes."
+        }
+      },
+      "description": "Specifications on number of machines."
+    },
+    "VirtualMachineNodesStatus": {
```

Comment:
What's the reasoning behind calling this "Status"? This seems to be a list of number of nodes and sizes, I don't see Status of nodes listed in the properties.